### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by pjsachdev** [casesandberg/react-color#678]
This is a really great project, I have enjoyed experimenting with it. Is anything being done regarding improving the general accessibility of the components for v3.0? If not, I would be happy to contribute.

**Comment by danrday** [comment]
@pjsachdev Any basic accessibility would be amazing- I for one would really appreciate it on this project. I haven't been able to find any open source color pickers for React that are accessible.

**Comment by pjsachdev** [comment]
@danrday Yeah, I struggled to find any open-source accessible color pickers too. I'm working on a PR for this project which fixes some basic accessibility issues that I have encountered. I'll keep you updated with the progress.

**Comment by danrday** [comment]
@pjsachdev  Awesome you rock!

**Comment by pjsachdev** [comment]
Sorry for the delay in working on this, but I've had a lot of other work recently. Here is my current progress on the accessibility improvements - https://github.com/pjsachdev/react-color/commits/a11y - there is still a lot to fix and improve. I would welcome any suggestions.
